mqtt/mqtt-broker/src/config.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Let's move this file with default settings to the `mqtt/mqtt-broker/config/default.json`

Reply:
Can we look at toml for this instead of json?

Reply:
I thought we had a discussion about this that part of the settings would go to twins, and because of that going with json is a smoother transition?

Reply:
ok makes sense.